org.wireshark.Wireshark.yml

```diff
@@ -14,6 +14,111 @@ finish-args:
 
 
 modules:
+  - name: libssh
+    cleanup:
+        - "/include"
+        - "/share"
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+        - type: archive
+          url: https://www.libssh.org/files/0.8/libssh-0.8.6.tar.xz
+          sha256: 1046b95632a07fc00b1ea70ee683072d0c8a23f544f4535440b727812002fd01
+
+  - name: c-ares
+    cleanup:
+        - "/include"
+        - "/share"
+    sources:
+        - type: archive
+          url: https://c-ares.haxx.se/download/c-ares-1.15.0.tar.gz
+          sha256: 6cdb97871f2930530c97deb7cf5c8fa4be5a0b02c7cea6e7c7667672a39d6852
+
+
+  - name: nghttp2
+    cleanup:
+        - "/include"
+        - "/share"
+    sources:
+        - type: archive
+          url: https://github.com/nghttp2/nghttp2/releases/download/v1.36.0/nghttp2-1.36.0.tar.xz
+          sha256: e9bb86157b88eda5a6844a232e039febbb52c1aa44b640acbbfabe729b8287fc
+
+
+  - name: lua52
+    disabled: true   # How to build lua5.2?
+    sources:
+    - type: archive
+      url: https://www.lua.org/ftp/lua-5.2.4.tar.gz
+      sha256: b9e2e4aad6789b3b63a056d442f7b39f0ecfca3ae0f1fc0ae4e9614401b69f4b
+
+
+  - name: libmaxminddb
+    cleanup:
+        - "/include"
+        - "/share"
+    sources:
+        - type: archive
+          url: https://github.com/maxmind/libmaxminddb/releases/download/1.3.2/libmaxminddb-1.3.2.tar.gz
+          sha256: e6f881aa6bd8cfa154a44d965450620df1f714c6dc9dd9971ad98f6e04f6c0f0
+
+        # Taken from https://github.com/flathub/net.openra.OpenRA/blob/master/net.openra.OpenRA.yaml
+        # I don't know whether we need this and if so, where to put those files
+        #- type: file
+        #  url: https://nuget.org/api/v2/package/MaxMind.Db/2.0.0
+        #  dest-filename: thirdparty/download/MaxMind.Db.zip
+        #  sha256: 98595abd21c735fe8bb84911a7b254f5d37c910818411f8edded7378e3e7c181
+
+        #- type: file
+        #  url: https://geolite.maxmind.com/download/geoip/database/GeoLite2-Country_20181211.tar.gz
+        #  dest: thirdparty/download/
+        #  sha256: 9e71cdb9cd870e1209001cbcfb0cb4dd3a58ff2d8ab85320682680aab0c250fc
+
+  - name: snappy
+    cleanup:
+        - /share/doc
+        - /include
+    sources:
+        - type: archive
+          url: https://github.com/google/snappy/releases/download/1.1.3/snappy-1.1.3.tar.gz
+          sha256: 2f1e82adf0868c9e26a5a7a3115111b6da7e432ddbac268a7ca2fae2a247eef3
+
+  - name: l4z
```

Comment:
Shouldn't this be "lz4" insteade of "l4z"?

Reply:
good call.